constants.js

```diff
@@ -308,6 +355,22 @@ const Errors = {
 			error: 'LSP6BatchExcessiveValueSent(uint256,uint256)',
 			message: 'LSP6: cannot send more `msg.value` than all the combined `values[]`.',
 		},
+		'0x6cb60587': {
+			error: 'NoCallsAllowed(address)',
+			message: 'LSP6: No calls allowed for the provided address',
+		},
+		'0x6fd203c5': {
+			error: 'InvalidWhitelistedCall(address)',
+			message: 'LSP6: Having all calls allowed in AllowedCalls is not allowed',
```

Comment:
Maybe we can use the word _whitelisting_ here to avoid repeating the word _allowed_? It also matches with the error name.
Not a big deal tho, it is still understandable.

```suggestion
			message: 'LSP6: Not allowed to whitelist all calls in AllowedCalls.',
```